src/components/Navbar.js

```diff
@@ -8,7 +8,7 @@ class Navbar extends Component {
 
     return (
       <>
-        <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
+        <nav className="navbar navbar-expand-lg navbar-dark bg-dark sticky-top">
```

Comment:
Sollten wir das nicht eher in dem `feature/navbar` branch machen?

Reply:
Man könnte ein hotfix/ machen, aber das ist so eine kleine Änderung, die kann man auch mitnehmen oder?